rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -303,6 +304,56 @@ def _get_db_cluster_status(db_service, expected_nodes_number):
     return db_service[STATUS]
 
 
+def _should_not_validate_cluster(service):
+    return service[STATUS] == ServiceStatus.FAIL or service[IS_EXTERNAL]
+
+
+def _get_broker_cluster_status(broker_service, expected_nodes_number):
+    """
+    Get the status of the broker cluster. The majority of nodes should be
```

Comment:
I will not document in comments, because this could change without updating these docs and the code should be clear enough